conda/base/context.py

```diff
@@ -249,7 +249,12 @@ def conda_in_private_env():
     # conda is located in its own private environment named '_conda'
     return basename(sys.prefix) == '_conda' and basename(dirname(sys.prefix)) == 'envs'
 
-context = Context(SEARCH_PATH, conda, None)
+try:
```

Comment:
Move this try/except block to the very bottom of the file.  It needs to have a more prominent place than buried in the middle (and it can't be at the beginning).